rdr_service/dao/genomics_dao.py

```diff
@@ -342,6 +343,21 @@ def get_member_from_sample_id(self, sample_id, genome_type):
             ).first()
         return member
 
+    def get_member_from_collection_tube(self, tube_id, genome_type):
+        """
+        Retrieves a genomic set member record matching the collection_tube_id
+        Needs a genome type
+        :param genome_type: aou_wgs, aou_array, aou_cvl
+        :param tube_id:
+        :return: a GenomicSetMember object
+        """
+        with self.session() as session:
+            member = session.query(GenomicSetMember).filter(
+                GenomicSetMember.collectionTubeId == tube_id,
+                GenomicSetMember.genomeType == genome_type
+            ).first()
```

Comment:
Is this always a "none or 1 result" expectation on the query?  Any reason to add a defensive check for multiple results?

Reply:
It should only ever be 1 or none. SqlAlchemy .first() doesn't return an error if None. SqlAlchemy .one() will raise an exception if no results are found, which I wasn't handling up the call stack. That may be something to add in a future PR.